Update to 1.20.1

spiralhalo opened the pull request:
Thanks for the cool mod.

Don't forget to [update JsonEM](https://github.com/FoundationGames/JsonEM/pull/13).

This even works on Forge!! And I never worked on Forge mods.

...well kinda. Creative tab ordering looks janky on Forge, but is otherwise functional. I don't know how to fix this.

Other relevant notes: 

Transportation tab has been removed in vanilla, and I can't think of a replacement tab, so Automobility items stay in their own tabs now.

spiralhalo committed:
Update to 1.20.1

Creative tab ordering looks janky on Forge, but is otherwise functional.

Transportation tab has been removed in vanilla. Automobility items
don't occupy any vanilla tab at the moment.

spiralhalo commented:
Please do take care of branch switching.

spiralhalo commented:
Death message works

![image](https://github.com/FoundationGames/Automobility/assets/8444172/a4c7d703-34f6-4e0a-af46-7105d4af4989)

FoundationGames committed:
Resolve conflicts, fix versions

FoundationGames committed:
Merge branch 'spiralhalo' into 1.20

FoundationGames commented:
Thanks for contributing! Looks good so far, will make a few important changes/bug fixes before merging soon

FoundationGames committed:
Add Parchment

FoundationGames committed:
Miscellaneous Fixes
- Slope placement fixed
- Auto assembler text has correct appearance
- Creative tabs are in the correct order
- Automobile death messages now function again
- Removed unused comments
